IdentityCore/src/cache/ios/MSIDKeychainTokenCache.m

```diff
@@ -349,6 +349,8 @@ - (BOOL)removeItemsWithKey:(MSIDTokenCacheKey *)key
         MSID_LOG_ERROR(context, @"Failed to delete keychain items (status: %d)", (int)status);
     }
     
+    [self saveWipeInfoWithContext:context error:nil];
```

Comment:
I think we shouldn't be saving Wipe data in here. We should only save Wipe data, when deleting a refresh token or when doing a wipe for user. I suggest we call saveWipeInfoWithContext from accessors.

Reply:
done :)